configuration/configuration_test.go

```diff
@@ -8,13 +8,15 @@ import (
 	"github.com/almighty/almighty-core/configuration"
 	"github.com/almighty/almighty-core/resource"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 const (
 	tDefaultConfigurationFile           = "config.yaml"
 	tEnvironmentVariableNameConfigFile  = "ALMIGHTY_CONFIG_FILE_PATH"
 	tEnvironmentVariableValueConfigFile = "../config.yaml"
 	tInvalidConfigFilePath              = "../invalid_config.yaml"
+	tTestConfigFilePath                 = "../test/data/multiple_config.yaml"
```

Comment:
Thanks for pointing out @kwk ,

My mistake, I missed using this - 
The objective of this file is to server as an alternate config file. 
The test `TestMultipleConfigurations()` consumes this and validates that we don't have a global config object anymore by comparing the values 'inside' `configurationData1` and `configurationData2` .